src/mono/sample/wasm/browser-eventpipe/Program.cs

```diff
@@ -1,162 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Why remove this? it should work with the threaded runtime

Reply:
reverted